### ISSUE TITLE
How do i get the classes folder to download

### ISSUE THREAD

**Issue by Dal1ster** [Haijo7/Online-Party-Chat#1]
When i ty to go to the folder or the file, it gives me the code, but not the file itself. What do i do?

**Comment by Haijo7** [comment]
When you click "code" then "download zip" it downloads the the mod.
![image](https://user-images.githubusercontent.com/67752550/104474159-0589db80-55be-11eb-857c-a2080d9c36a5.png)

**Comment by Haijo7** [comment]
I don't think you get a notification from that unless I do @Dal1ster, so this should probably notify you.
